.github/workflows/biome.yml

```diff
@@ -0,0 +1,43 @@
+name: "biome"
+
+on:
+  workflow_call:
+    inputs:
+      head_sha:
+        required: true
+        type: string
+
+env:
+  NODE: "18"
+  FRONTEND_MONOREPO_DIR: "web"
+
+jobs:
+  main:
+    name: "biomejs"
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: hmarr/debug-action@v3.0.0
+
+      - name: Configure git
+        shell: bash
+        run: |
+          set -xeuo pipefail
+          git config --global user.name 'robot-ci-heartex'
```

Comment:
Will it push changes into branch back?
If no - it's not required

Reply:
I try to stick to the pattern I found in the platform, although I may have chosen the wrong one. Thank you for the review. This PR is mostly generated through a script and I will re-run the script on the day of the release to ensure that biome is applied to any new changes. 

Should I separate this for this PR and create another one, where biome is not required, but already in the platform? And in this PR, should I only do the required part?

Reply:
> Will it push changes into branch back?
My only intention is to block the pr not alter the the branch

Reply:
Done!